lib/action_tracker/base.rb

```diff
@@ -1,19 +1,20 @@
 module ActionTracker
   # nodoc
   class Base
-    attr_reader :kontroller, :resource
+    attr_reader :controller, :resource
+    alias_method :kontroller, :controller
```

Comment:
Why is it necessary ? If ```kontroller``` is going to be removed, we should expect the users to update its calls based on a version upgrade, since its well documented. What do you think?